Neos.Neos/NodeTypes/Mixin/Content.yaml

```
@@ -3,6 +3,8 @@
     'Neos.Neos:Node': true
     'Neos.Neos:Hidable': true
   abstract: true
+  strategy:
+    moveNode: RelationDistributionStrategy::STRATEGY_SCATTER
```

I think _that's_ the root cause. Why not
```suggestion
    moveNode: scatter
```

---

This format is what we decided during the sprint when discussing the new setting. It is a lot clearer than only using the key

---

The thing is: If it is clearer, we should change the values of those enums to reflect this, i.e. change
```php
case STRATEGY_SCATTER = 'scatter';
```
to 
```php
case STRATEGY_SCATTER = 'STRATEGY_SCATTER';
```

(like we did for many other enums exactly to avoid that confusion).

But without that change, having a different serialization format wether it's in the database (or a JSON representation) from the YAML representation is just error prone in my eyes.

Anyways, I don't want to block this PR of course – feel free to ignore me, just my 2ct :)